src/Microsoft.Azure.WebJobs.Host/Timers/IWebJobsExceptionHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Timers
+{
+    /// <summary>
+    /// Represents an unhandled exception handler.
+    /// </summary>
+    public interface IWebJobsExceptionHandler
+    {
+        /// <summary>
+        /// Called during host creation. Allows a reference to the host.
+        /// </summary>
+        /// <param name="host">The JobHost.</param>
+        void Initialize(JobHost host);
```

Comment:
That's what I'd done originally -- but then I realized that other implementors may want access to this as well and there's no way for them to get it.

Maybe I was overthinking it?

---
In reply to: [77049998](https://github.com/Azure/azure-webjobs-sdk/pull/799#discussion_r77049998) [](ancestors = 77049998)